build.sc

```diff
@@ -662,12 +667,12 @@ def publishExecutable() = {
     println("MASTER COMMIT: Creating a release")
     if (!unstable){
       requests.post(
-        "https://api.github.com/repos/lihaoyi/Ammonite/releases",
+        s"https://api.github.com/repos/${ghOrg}/${ghRepo}/releases",
         data = ujson.write(
           ujson.Obj(
             "tag_name" -> buildVersion,
             "name" -> buildVersion,
-            "body" -> s"http://www.lihaoyi.com/Ammonite/#$buildVersion"
+            "body" -> s"${homePage}/#${buildVersion}"
```

Comment:
Will this have double slash? Because `homePage` has a trailing slash.  
I prefer to have base URLs without a trailing slash..

Reply:
Yeah, makes sense. Me too.